includes/Admin/Product_Categories.php

```diff
@@ -143,10 +148,30 @@ public function get_google_product_category_field_title() {
 	 *
 	 * @since 2.1.0-dev.1
 	 *
-	 * @return string
+	 * @param int $term_id term ID
+	 * @param int $tt_id term taxonomy ID
+	 * @param string $taxonomy Taxonomy slug
 	 */
-	public function save_google_product_category() {
+	public function save_google_product_category( $term_id, $tt_id, $taxonomy ) {
+
+		$google_product_category_id = Framework\SV_WC_Helper::get_posted_value( self::FIELD_GOOGLE_PRODUCT_CATEGORY_ID );
```

Comment:
I think we should clean the value and perhaps ensure it is a numeric string.

```suggestion
		$google_product_category_id = wc_clean( Framework\SV_WC_Helper::get_posted_value( self::FIELD_GOOGLE_PRODUCT_CATEGORY_ID ) );
```